aspnetcore/security/authorization/resourcebased.md

```diff
@@ -60,13 +70,13 @@ In the following example, the resource to be secured is loaded into a custom `Do
 
 ::: moniker range=">= aspnetcore-2.0"
 
-[!code-csharp[](resourcebased/samples/ResourceBasedAuthApp2/Pages/Document/Edit.cshtml.cs?name=snippet_DocumentEditHandler)]
+[!code-csharp[](resourcebased/samples/3_0/ResourceBasedAuthApp2/Pages/Document/Edit.cshtml.cs?name=snippet_DocumentEditHandler)]
```

Comment:
The 3.0 folder doesn't have Pages. This path is incorrect (see the build report from the details tab)

Reply:
fixed, thanks